### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by ADmad**
Only run DatabaseSuite for non-mysql to speed up tests.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -73,8 +73,11 @@ before_script:
   - composer install --prefer-dist --no-interaction
 
 script:
-  - if [[ $DEFAULT = 1 && $TRAVIS_PHP_VERSION = 7.0 ]]; then export CODECOVERAGE=1; vendor/bin/phpunit --coverage-clover=clover.xml; fi
-  - if [[ $DEFAULT = 1 && $TRAVIS_PHP_VERSION != 7.0 ]]; then vendor/bin/phpunit; fi
+  - if [[ $DEFAULT = 1 && $TRAVIS_PHP_VERSION = 7.0 && $DB = 'mysql' ]]; then export CODECOVERAGE=1; vendor/bin/phpunit --coverage-clover=clover.xml; fi
+  - if [[ $DEFAULT = 1 && $TRAVIS_PHP_VERSION = 7.0 && $DB != 'mysql' ]]; then export CODECOVERAGE=1; vendor/bin/phpunit tests/TestCase/DatabaseSuite.php --coverage-clover=clover.xml; fi
+
+  - if [[ $DEFAULT = 1 && $TRAVIS_PHP_VERSION != 7.0 && $DB = 'mysql' ]]; then vendor/bin/phpunit; fi
+  - if [[ $DEFAULT = 1 && $TRAVIS_PHP_VERSION != 7.0 && $DB != 'mysql' ]]; then vendor/bin/phpunit tests/TestCase/DatabaseSuite.php; fi
```
Comment (antograssiot) [inline .travis.yml]: Could't yiu use the  `--testsuite` option here ?

**Comment by ADmad** [comment]
Hmm codecov is not impressed. Let me see if I can coverage to previous level else I'll close the PR.

**Comment by thinkingmedia** [comment]
Some of the tests must not be grouped correctly.

**Comment by ADmad** [comment]
@thinkingmedia Maybe, but I am not inclined to review the test cases and find out :)

**Pull request closed without merging**